src/components/Paragraph/paragraph.scss

```diff
@@ -5,6 +5,9 @@
 }
 
 .no-margin {
-  composes: root;
   margin-bottom: 0;
 }
+
+.small {
```

Comment:
Surely here you might use a normal `<p>` element instead of `<Paragraph>` ?
I think base `<p>` has `font-size` of `1em` and no margin